examples/DISCO_IOT_Record/DISCO_IOT_Record.ino

```diff
@@ -0,0 +1,109 @@
+/**
+ ******************************************************************************
+ * @file    DISCO_IOT_Record.ino
```

Comment:
This sample is linked to https://github.com/stm32duino/USBMicrophone repo. It would be good to at least mention this here. (or even move the example to USBMicrophone repo ?)

An other example can be a simple record, in a buffer, in a SD ?

Reply:
I added simple example and remove USB example. USBMircrophone doesn't work anymore.